ovs/dal/hybrids/vdisk.py

```diff
@@ -296,3 +302,15 @@ def calculate_delta(key, dynamic, current_stats):
                 else:
                     current_stats['{0}_ps'.format(key)] = max(0, (current_stats[key] - previous_stats[key]) / delta)
         volatile.set(prev_key, current_stats, dynamic.timeout * 10)
+
+    def reload_clients(self, client):
```

Comment:
Function name does not cover the load
A rename was unnecessary imo, since you only reload 1 type of client, so ```reload_client``` was just fine